docs/src/main/paradox/directives/index.md

```diff
@@ -12,6 +12,7 @@ More directives are available via @ref[extensions](../customization/extensions.m
 
  * [Index and Table Of Contents Directives](organizing-pages.md)
  * [Linking Directives](linking.md)
+ * [dependencies](dependencies.md)
```

Comment:
```suggestion
 * [Dependencies](dependencies.md)
```

Reply:
As that text doesn't show I intentionally deviate in what I write.